### PR TITLE
add global log level option for nydusify

### PR DESCRIPTION
**Pull request by liubin**
Add --log-level option and -D as global options for
pack/convert/check subcommands.

The -D is the shortcut for --log-level debug,
and will have the highest priority.

Signed-off-by: bin liu <bin@hyper.sh>

### CODE REVIEW

File: contrib/nydusify/cmd/nydusify.go [hunk 1]
```diff
@@ -528,3 +539,19 @@ func main() {
...
+	logLevel, err := logrus.ParseLevel(lvl)
+	if err != nil {
+		logrus.Warnf("failed to parse log level(%s): %+v\ndefault log level(info) will be used", lvl, err)
+		logLevel = defaultLogLevel
+	}
+
+	logrus.SetLevel(logLevel)
+
+	// global `-D` has the highest priority
+	if c.Bool("D") {
```
Comment (imeoer) [inline contrib/nydusify/cmd/nydusify.go:554]: We can check this at the beginning of the func.

Comment (liubin) [reply contrib/nydusify/cmd/nydusify.go:554]: Check it last to let it overwrite the `--log-level`.

Comment (imeoer) [reply contrib/nydusify/cmd/nydusify.go:554]: How about this:

```
// global `-D` has the highest priority
if c.Bool("D") {
	return logrus.SetLevel(logrus.DebugLevel)
}

lvl := c.String("log-level")
logLevel, err := logrus.ParseLevel(lvl)
if err != nil {
	logrus.Warnf("failed to parse log level(%s): %+v\ndefault log level(info) will be used", lvl, err)
	logLevel = defaultLogLevel
}
logrus.SetLevel(logLevel)
```

Comment (liubin) [reply contrib/nydusify/cmd/nydusify.go:554]: Or which should have the higher priority?

**Review by imeoer — APPROVED** [review]
LGTM!

**Pull request merged**